CMS-Enterprise/mint-app
# [MINT-3210] Custom Milestone Form Modal

# MINT-3210

## Description


* Added `MilestoneForm` component and test and copy
* Added `CreateMTOMilestoneCustom`
* Added temporary button section for the Custom Milestone Modal in `ModelToOperations/Home/index.tsx`



## How to test this change

1.  navigate to MTO Home
2. Click on `Add Custom Milestone` button
3. Verify that the design matches figma
4. Select Primary Category
5. Verify that the subcategories listed are the ones expected to show for whatever primary category that's selected
6. See success or error alerts to match figma


## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
